frontend/src/components/uam/helpers/get-columns-helper/get-users-columns.helper.ts

```diff
@@ -20,6 +23,12 @@ const getUsersColumns = (): Column<UsersGetResponseDto>[] => {
       Header: 'Created',
       accessor: UserTableAccessor.CREATED_AT,
     },
+    {
+      Header: 'Actions',
+      accessor: UserTableAccessor.ACTIONS_PAYLOAD,
+      Cell: (cell) =>
+        Actions(String(cell.row.original.actions_payload), onUserDelete),
+    },
```

Comment:
It may work this way, then look at how to get values from normal props for Actions component
```suggestion
      accessor: (row: UsersTableType): ActionsCellProps => ({
          userId: number,
          onUserDelete
      }),
      Cell: Actions,
    },
```

Reply:
agreed